pollen_vision/pollen_vision/perception/perception.py

```diff
@@ -120,33 +160,33 @@ def set_tracked_objects(self, objects: list[str]) -> None:
                 self.tracked_objects.append(obj)
 
 
-if __name__ == "__main__":
-    import FramesViewer.utils as fv_utils
-    from pollen_vision.camera_wrappers.depthai import SDKWrapper
-    from pollen_vision.camera_wrappers.depthai.utils import get_config_file_path
+# if __name__ == "__main__":
```

Comment:
Why comment this ? Is this code not working anymore ?

Reply:
my mistake :)